ch_tools/chadmin/cli/table_group.py

```diff
@@ -167,6 +175,8 @@ def columns_command(ctx, database_name, table_name):
 
 
 @table_group.command("delete")
+@argument("database_name", metavar="DATABASE", required=False)
```

Comment:
Suggest to implement support of database name and table name args for non-detached tables.

Reply:
I suggest to do it in another PR, if you don't mind.